frontend/src/HomePage/SignHandlers/SignUp.css

```diff
@@ -0,0 +1,60 @@
+.topbar {
+    display: block;
+}
+
+.signupview {
+    z-index: 100;
+    font-family: "Inter Tight", sans-serif;
+    margin-top: 95px;
+    font-weight: 400;
+    background-color: #fefefe ;
```

Comment:
One good habit to get into early in the project is to perhaps have a single place where you put all colors (organized semantically eg. primary, secondary, background (rather than blue, red, green, yellow etc.)). That makes it easier to do wholistic changes in the project

Reply:
This is a good point and potentially something to include in your wireframes!